css-positioning/absolute.css

```diff
@@ -0,0 +1,41 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+
```

Comment:
you need to have the proper position property assigned here

Reply:
Updated